.github/actions/build-zui/action.yml

```diff
@@ -47,16 +51,45 @@ runs:
         security find-identity -p codesigning -v
       shell: bash
 
+    - name: Checkout esigner-codesign repository
+      if: runner.os == 'Windows'
+      uses: actions/checkout@v3
+      with:
+       repository: 'SSLcom/esigner-codesign'
+       path: esigner-codesign
+
+    - name: Make values from package.json available in Actions steps
+      id: zui-package
+      uses: RadovanPelka/github-action-json@v1.0.1
+      with:
+        path: apps/zui/package.json
+
     - name: Build & Publish
       run: ${{ inputs.cmd }}
       shell: bash
       env:
         GH_TOKEN: ${{ inputs.gh_token }}
-        WIN_CSC_KEY_PASSWORD: ${{ inputs.csc_key_password }}
-        WIN_CSC_LINK: ${{ inputs.csc_link }}
         APPLE_ID: ${{ inputs.apple_id }}
         APPLE_ID_PASSWORD: ${{ inputs.apple_id_password }}
         APPLE_TEAM_ID: ${{ inputs.apple_team_id }}
+        CODE_SIGN_SCRIPT_PATH: ${{ github.workspace }}/esigner-codesign/dist/index.js
+        INPUT_COMMAND: sign
+        INPUT_FILE_PATH: ${{ github.workspace }}/dist/apps/zui/${{ fromJSON(steps.zui-package.outputs.productName) }} Setup ${{ fromJSON(steps.zui-package.outputs.version) }}.exe
+        INPUT_OVERRIDE: true
+        INPUT_MALWARE_BLOCK: false
+        INPUT_CLEAN_LOGS: false
+        INPUT_JVM_MAX_MEMORY: 1024M
+        INPUT_ENVIRONMENT_NAME: PROD
+        INPUT_USERNAME: ${{ inputs.ssl_com_username }}
+        INPUT_PASSWORD: ${{ inputs.ssl_com_password }}
+        INPUT_TOTP_SECRET: ${{ inputs.ssl_com_totp_secret }}
+        INPUT_CREDENTIAL_ID: ${{ inputs.ssl_com_credential_id }}
+
+    - name: Check for successful signing with SignTool
+      if: runner.os == 'Windows'
+      run: |
+        "C:\Program Files (x86)\Microsoft SDKs\ClickOnce\SignTool\signtool.exe" verify /pa "${{ github.workspace }}/dist/apps/zui/${{ fromJSON(steps.zui-package.outputs.productName) }} Setup ${{ fromJSON(steps.zui-package.outputs.version) }}.exe"
+      shell: cmd
```

Comment:
While getting the esigner tool working with electron-builder, one buggy-seeming thing I encountered is that even if the custom signing code returns `false` due to an error, the electron-builder apparently still behaved as if it succeeded. [Here's an example run](https://github.com/brimdata/zui/actions/runs/8715359091/job/23907131337) where you can see what happens: At first it looks green/successful, but if you expand the "Build Zui" step and scroll to the bottom you can see the red failure message. Bad, right?

This would have made it likely that we could one day be producing unsigned builds and not be aware. I found via web searches that this `signtool verify /pa` approach adequately sniffs out if an executable is, indeed, signed. I tested it out and it does what we need ([example run catching an unsigned build](https://github.com/brimdata/zui/actions/runs/8728269511)).